submissions/nadsatt/html-css-popup/index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>popup</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <nav class="nav">
+            <ul class="list">
+                <li class="list__item">
+                    <a class="list__link" href="#">Mail</a>
+                </li>
+                <li class="list__item">
+                    <a class="list__link" href="#">Images</a>
+                </li>
+                <li class="list__item">      
+
+                    <input class="list__check-input check-input" id="list-check-input" type="checkbox">
+                    <label class="list__check-label list__link" for="list-check-input">
+                       <svg class="list__check-icon list__icon" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="24" height="24" viewBox="0 0 512 512" enable-background="new 0 0 512 512" xml:space="preserve">
+                            <path d="M186.2,139.6h139.6V0H186.2V139.6z M372.4,0v139.6H512V0H372.4z M0,139.6h139.6V0H0V139.6z M186.2,325.8h139.6V186.2H186.2 V325.8z M372.4,325.8H512V186.2H372.4V325.8z M0,325.8h139.6V186.2H0V325.8z M186.2,512h139.6V372.4H186.2V512z M372.4,512H512 V372.4H372.4V512z M0,512h139.6V372.4H0V512z"/>
+                        </svg>
+                    </label>
+ 
+                    <div class="list__popup">
+                        <div class="popup__top-container">
```

Comment:
In this case, we have BEM element `popup__top-container`, but I can't see a parent block `popup`.
Please take a look at this issue.

Reply:
Done